app/routes/signin-oidc.js

```diff
@@ -115,6 +115,21 @@ function getRedirectPath(
   }
 }
 
+function safelyGetLoginSource(request) {
```

Comment:
```suggestion
export const safelyGetLoginSource = (request) => {
```

Reply:
Just to match the style of other functions

Reply:
not sure it needs exporting though